pkg/rpc/server_test.go

```diff
@@ -248,7 +247,7 @@ func TestHandler(t *testing.T) {
 
 	store, err := storage.NewLevelDBStore(cfg.ApplicationConfiguration.DBConfiguration.LevelDBOptions)
 	assert.Nil(t, err)
-	chain, err := core.NewBlockchain(context.Background(), store, cfg.ProtocolConfiguration)
+	chain, err := core.NewBlockchain(store, cfg.ProtocolConfiguration)
```

Comment:
in this test there's no need to chain.Run() since blockchain reads all the data only once and tests don't expect to have updated/changed chains. That's not good or not bad. It is what it is. Next PR will be related to reworking server_test.go